### PR TITLE
fix readme link

### PR DESCRIPTION
**Pull request by k-nasa**
**Choose one:** is this a 🐛 bug fix, a 🙋 feature, or a 🔦 documentation change?

## Checklist

- [ ] tests pass
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,8 @@ $ cargo add memdb
 [1]: https://img.shields.io/crates/v/memdb.svg?style=flat-square
 [2]: https://crates.io/crates/memdb
 [3]: https://img.shields.io/travis/rustasync/memdb.svg?style=flat-square
```
Comment (taiki-e) [inline README.md]:
```suggestion
 [3]: https://travis-ci.com/rustasync/memdb.svg?branch=master
```

File: README.md [hunk 2]
```diff
@@ -28,8 +28,8 @@ $ cargo add memdb
...
-[4]: https://travis-ci.org/rustasync/memdb
-[5]: https://img.shields.io/crates/d/memdb.svg?style=flat-square
+[4]: https://travis-ci.com/rustasync/memdb
+[5]: https://travis-ci.com/rustasync/memdb.svg?branch=master
```
Comment (taiki-e) [inline README.md]:
```suggestion
[5]: https://img.shields.io/crates/d/memdb.svg?style=flat-square
```

(`[5]` is crate.io badge)

**Comment by k-nasa** [comment]
@taiki-e thx

**Review by taiki-e — APPROVED** [review]
Thanks

**Pull request merged**